dragonchain/webserver/routes/smart_contracts.py

```diff
@@ -35,6 +35,8 @@ def apply_routes(app: flask.Flask):
     app.add_url_rule("/v1/contract", "list_contract_v1", list_contract_v1, methods=["GET"])
     app.add_url_rule("/contract/<contract_id>", "get_contract_by_id_v1", get_contract_by_id_v1, methods=["GET"])
     app.add_url_rule("/v1/contract/<contract_id>", "get_contract_by_id_v1", get_contract_by_id_v1, methods=["GET"])
+    app.add_url_rule("/contract/logs/<contract_id>", "get_smart_contract_logs_v1", get_smart_contract_logs_v1, methods=["GET"])
```

Comment:
don't need the unversioned route for new endpoints. Only reason those exist on other routes is for legacy support.